source/common/res/features/hide-help/main.js

```diff
@@ -3,9 +3,19 @@
   if (typeof ynabToolKit !== 'undefined' && ynabToolKit.pageReady === true) {
     ynabToolKit.hideHelp = (function () {
       return {
+        hideHelp: 'true',
+
         invoke() {
-          // hide button by default
-          $('#hs-beacon').hide();
+          let hideHelp = ynabToolKit.shared.getToolkitStorageKey('hide-help');
```

Comment:
You can specify type in the second parameter here

`ynabToolKit.shared.getToolkitStorageKey('hide-help', 'boolean')`